frontend/appflowy_flutter/packages/flowy_infra/lib/colorscheme/colorscheme.dart

```diff
@@ -128,6 +129,7 @@ class FlowyColorScheme {
     required this.topbarBg,
     required this.icon,
     required this.text,
+    required this.textAlt,
```

Comment:
This seems redundant now

Reply:
I tried for directly choosing black or white colour according to contrast but it did not give satisfactory results in all themes and lightings. So i added manually two texts themes : Dark and White colour.